databases/week2/Library_Mgmt_System_DB.sql

```diff
@@ -0,0 +1,72 @@
+-- databses/week2 Homework
+-- PART 4 : Creating a database: Using an entity relationship diagram, 
+-- design the data model for an application of your choice
+-- Iam designing a database model for a simple library management system
+-- for this i need following tables and the relationships between them
+use LibraryMgmtSystemDB;
+-- each book has a unique id, members of the library can search books by title, author, category or publication year
+CREATE TABLE `book` (
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `title` VARCHAR(100) NOT NULL,
+    `author` VARCHAR(255) NOT NULL,
+    `category` VARCHAR(255) NOT NULL,
+    `publication_year` DATE NOT NULL
+);
+-- a book can have multiple authors
+CREATE TABLE `author` (
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `first_name` VARCHAR(150) NOT NULL,
+    `last_name` VARCHAR(150) NOT NULL
+);
```

Comment:
You should specify the character set, see [here](https://github.com/HackYourFuture-CPH/databases/blob/master/lesson2/lesson2-data.sql) for an example. Otherwise i'm guessing that if you have names written in chinese or arabic all you're going to see is `???`.

Same for all other tables in the homework ;p